mgmt_asset/demo/demo.xml

```diff
@@ -1,7 +1,17 @@
 <odoo>
 
-    <record id="demo_asset0" model="mgmt.asset">
-        <field name="name">Asset 0</field>
+    <record id="demo_inventory_category_servers" model="maintenance.equipment.category">
+        <field name="name">Servers</field>
+    </record>
+
+    <record id="demo_inventory_server" model="maintenance.equipment">
+        <field name="name">Odoo-Server</field>
+        <field name="category_id" ref="demo_inventory_category_servers" />
+    </record>
+
+    <record id="demo_asset_server" model="mgmt.asset">
```

Comment:
Setting up the asset might be difficult. Check if you can use `eval="ref('mgmt_asset.demo_inventory_server').id"` to get the record id.

I updated the ERD. The model_name is required and model_id is computed from model_name.